Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Action/ActionManagerInterface.h

```diff
@@ -47,6 +47,17 @@ namespace AzToolsFramework
         bool m_hideFromToolBarsWhenDisabled = false;
     };
 
+    struct WidgetActionProperties
```

Comment:
suggestion: API comment here would likely be useful (e.g. what the properties are for and where they are used)